src/main/java/co/inventorsoft/StreamHomework.java

```diff
@@ -29,7 +34,7 @@ public List<Person> extractTeenagers(final List<Person> people) {
      * @return collection of user, without duplicates
      */
     public List<User> createUsers(final List<String> emails) {
-        return null;
+        return emails.stream().filter(s -> s != null).distinct().map(User::new).collect(Collectors.toList());
```

Comment:
For the future: you can make use of `Objects::nonNull` method reference in such a case. There's no need to change it now, just keep in mind:)